shambomon/lib/shambomon/game.ex

```diff
@@ -141,6 +144,11 @@ defmodule Shambomon.Game do
       game = update_messages(game, Map.get(p2, :char), health_decr)
     end
 
+    # Set the gameOver flag if applicable
+    if (Map.get(p1, :health) <= 0) or Map.get(p2, :health <= 0) do
```

Comment:
This is where the `gameOver` flag is set.

Reply:
Parenthesis was in the wrong spot on that second condition, so the bug occurs when p2 dies. rip